package.json

```diff
@@ -1,32 +1,46 @@
 {
-  "name": "d2l-image",
   "description": "A Polymer-based web component for images requiring authentication",
-  "private": true,
-  "scripts": {
-    "postinstall": "polymer install --variants",
-    "test": "npm run lint && polymer test --skip-plugin sauce",
-    "lint": "npm run lint:js && npm run lint:wc",
-    "lint:js": "eslint *.html",
-    "lint:wc": "polymer lint -i d2l-image.html"
-  },
-  "homepage": "https://github.com/Brightspace/d2l-image#readme",
+  "keywords": [
+    "d2l",
+    "image"
+  ],
   "repository": {
     "type": "git",
     "url": "git+https://github.com/Brightspace/d2l-image.git"
   },
+  "homepage": "https://github.com/Brightspace/d2l-image#readme",
+  "name": "d2l-image",
+  "scripts": {
+    "test": "npm run lint && polymer test --module-resolution=node --npm --skip-plugin sauce",
+    "lint": "npm run lint:js && npm run lint:wc",
+    "lint:js": "eslint *.js",
+    "lint:wc": "polymer lint -i d2l-image.js"
+  },
   "bugs": {
     "url": "https://github.com/Brightspace/d2l-image/issues"
   },
-  "keywords": [
-    "d2l",
-    "image"
-  ],
   "author": "D2L Corporation",
   "license": "Apache-2.0",
   "devDependencies": {
+    "@webcomponents/webcomponentsjs": "^2.0.2",
+    "babel-eslint": "^8.2.3",
     "eslint": "^4.15.0",
     "eslint-config-brightspace": "^0.4.0",
     "eslint-plugin-html": "^4.0.1",
-    "polymer-cli": "^1.5.7"
+    "polymer-cli": "^1.7.3",
+    "wct-browser-legacy": "^1.0.1"
```

Comment:
Why are they calling it "legacy"? Is there a new WCT they're working on that isn't ready yet?

Reply:
I was also confused by this - couldn't figure that out with 60 or so seconds of googling 😐

Reply:
It's part of the modulizer conversion, and if this package doesn't exist, `polymer test` fails with a mysterious error.